project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
 
-  final val Scala_2_13 = "2.13.3"
+  final val Scala_2_13 = "2.13.0"
```

Comment:
This sucks, but I think it is related to https://github.com/typelevel/kind-projector/issues/116